resources/functionMap.php

```diff
@@ -996,8 +996,8 @@
 'closelog' => ['bool'],
 'Closure::__construct' => ['void'],
 'Closure::__invoke' => ['', '...args='=>''],
-'Closure::bind' => ['Closure', 'old'=>'Closure', 'to'=>'?object', 'scope='=>'object|string|null'],
-'Closure::bindTo' => ['Closure', 'new'=>'?object', 'newscope='=>'object|string|null'],
+'Closure::bind' => ['Closure', 'old'=>'Closure', 'to'=>'?object', 'scope='=>'object|class-string|null'],
+'Closure::bindTo' => ['Closure', 'new'=>'?object', 'newscope='=>'object|class-string|null'],
```

Comment:
We typically want to narrow parameter types only in bleedingEdge. This change should be done in resources/functionMap_bleedingEdge.php.

That's because this is basically a BC break. You're forcing users that pass a `string` in there to add/rewrite their PHPDoc types.

Reply:
done